ui/packages/catalog-ui-search/src/main/webapp/component/search-form-list/search-form-list.view.js

```diff
@@ -50,8 +77,11 @@ class SearchForms extends React.Component {
     const { filter } = this.state
     const { forms, onClick } = this.props
 
+    const filteredForms = forms
+      .filter(form => form.title.toLowerCase().match(filter.toLowerCase()));
```

Comment:
would it make sense to use [this](https://github.com/codice/ddf/pull/4254/files#diff-cef0db9c2e7d4b980b769ae0092ded3aR15) filtering logic here?

Reply:
oooo good idea!

Reply:
I looked into it but I think I like the current implementation better. It's a lot more forgiving than the `matchFilter` approach. For example, if you had some forms named "dogcat" and "catdog" and you search for "dog", the current approach will match both but `matchFilter` will only match "dogcat"

Reply:
IMO we should be consistent and matchFilter makes sense. Your `dogcat` example sounds right because those really are two words. Consider a search for `GRE`, do we really want `PROGRESS FORM` to be one of the search results?

Reply:
unless we think people are going to nameformslikethis

Reply:
Nice counter example, @rymach . I like the current approach because it wouldn't matter how they name it, but I do see how `EXAMPLE SEARCH FORM` could potentially just be noise in the filtered results if they are searching for `ORM`. And consistency is 💯 so I'll change it to use `matchFilter`.